tests/unit_tests/test_wave_06.py

```diff
@@ -109,8 +110,12 @@ def test_swap_best_by_category():
         my_priority="Clothing",
         their_priority="Decor"
     )
-
-    raise Exception("Complete this test according to comments below.")
+    assert result is True
+    assert len(jesse.inventory) == 3
+    assert len(tai.inventory) == 3
+    assert item_f in tai.inventory
+    assert item_c in jesse.inventory
```

Comment:
I recommend checking that *all* the items are correct, since the instructions say:

>That all the correct items are in tai and jesse's inventories, including the items which were swapped from one vendor to the other